ruby.example/fragments/methods.aterm

```diff
@@ -0,0 +1,46 @@
+Program(
```

Comment:
I would remove this file and update the gitignore file to ignore all aterm files.